src/Client/Web/Bit.Client.Web.BlazorUI/Components/Inputs/BitTextField.razor.cs

```diff
@@ -0,0 +1,172 @@
+﻿using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Web;
+using System.Threading.Tasks;
+
+namespace Bit.Client.Web.BlazorUI
+{
+    public partial class BitTextField
+    {
+        private bool _isMultiLine = false;
+
+        [Parameter] public int MaxLength { get; set; } = -1;
+        [Parameter] public string Value { get; set; }
+        [Parameter] public string Placeholder { get; set; }
+        [Parameter] public bool IsReadonly { get; set; } = false;
+        [Parameter] public TextFieldType Type { get; set; } = TextFieldType.Text;
+        [Parameter]
+        public bool IsMultiLine
+        {
+            get => _isMultiLine;
+            set
+            {
+                if (Type == TextFieldType.Text)
```

Comment:
This check is not required, it's just a property setter and should not be conditional.